src/modules/blocks/rsvp/style.pcss

```
@@ -40,48 +40,38 @@
 }
 
 .tribe-editor__inactive-block--rsvp {
-	.tribe-editor__inactive-block--rsvp__container {
-		background-color: var(--tec-color-background);
-		border: 1px solid var(--tec-color-border-secondary);
-		border-radius: var(--tec-border-radius-default);
-		margin-top: var(--tec-spacer-6);
-		max-width: 576px;
-		overflow: hidden;
-		position: relative;
-		transition: all 0.3s ease;
-		width: 100%;
-	}
-
-	.tribe-editor__rsvp {
-		padding: var(--tec-spacer-10) var(--tec-grid-gutter-small-half);
-	}
+	display: flex;
+	justify-content: space-between;
+	flex-direction: row;
+	padding: var( --tec-spacer-10 ) 0;
```

Styling properties should be in alphabetical order.
```suggestion
	display: flex;
	flex-direction: row;
	justify-content: space-between;
	padding: var( --tec-spacer-10 ) 0;
```

---

thanks! This is fixed now.